src/app/providers/LocalConfigContext.ts

```diff
@@ -0,0 +1,16 @@
+import { createContext } from 'react'
+import { TableAgeType } from '../../types/table-age-type'
+
+/**
+ * This is where we store local state
+ */
+export interface LocalConfigProviderState {
+  ageHeaderType: TableAgeType
```

Comment:
Also here, if we would start adding more options. I would prefer it would be a single local storage key and it being stored as a JSON of options( `{[option-key]: option-value }`), opposed to creating new local storage for each of the option.

Reply:
>  I would prefer it would be a single local storage key and it being stored as a JSON of options

Reorganized accordingly